src/components/Account/ButtonLogin.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { useAuth0 } from '@auth0/auth0-react';
+
+
+const ButtonLogin = () => {
+    const {loginWithRedirect} =  useAuth0()
+  return (
+    <>
+    <button onClick={()=>loginWithRedirect()}> Account </button>
```

Comment:
como aun no esta autenticado deberia decir
Account  => Sign In

Puede haber otro boton que diga Log In

Sign In (Crear cuenta)

Log In (Iniciar sesion)